### ISSUE TITLE
Support forever scope

### ISSUE THREAD

**Issue by bilelmoussaoui** [gtk-rs/gir#1372]
Currently gir fails with when trying to generate Pango2 (1.90 currently) bindings
```
Error: Error: "GirXml: pango2/sys/../../Pango2-1.0.gir at line 1775:11: Attribute `scope` on element <parameter> has invalid value: Unknown parameter scope type: forever"
```

**Comment by sdroege** [comment]
Yeah that's an easy one. GStreamer also has a couple of those but not annotated accordingly and manually bound.

**Comment by bilelmoussaoui** [comment]
Do you remember which ones that needs to be fixed so I can try my changes?

**Comment by sdroege** [comment]
They will continue to stay manual because of other issues. `gst_tag_register()` for example.

**Comment by Vanadiae** [comment]
I also tried generating pango2 bindings and encountered the same issue. After some grepping and code reading it looks like the code that need to be changed live in `library.rs` and `analysis/rust_type.rs`, but apart from that I've no knowledge about how gir works. For now I've replaced the "forever" scopes with "call" even if that's wrong, but I won't use `pango2_attr_type_register()` for now so that's not so much of an issue.
